packages/dd-trace/src/telemetry/index.js

```diff
@@ -312,12 +317,18 @@ function updateConfig (changes, config) {
   }
 
   const namesNeedFormatting = new Set(['DD_TAGS', 'peerServiceMapping'])
+  const entriesNotSentOnUndefinedDefault = new Set(['appsec.sca.enabled'])
 
   const configuration = []
   const names = [] // list of config names whose values have been changed
 
   for (const change of changes) {
     const name = nameMapping[change.name] || change.name
+
+    if (entriesNotSentOnUndefinedDefault.has(name) && change.origin === 'default' && change.value === undefined) {
+      continue
+    }
```

Comment:
wow this is so dirty, we shouldn't need specific hardcoded rules like that just because the backend asks us to do it, we send the default undefined for all other configs...